uint8 id
geometry_msgs/Pose[] reachable_poses
geometry_msgs/Pose initial_pose